src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,58 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: (tab: Tab) => void,
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const currentTab = tabs
+    .some((searchTab) => searchTab.id === selectedTabId);
+
+  return (
+    <section data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map((tab: Tab, index: number) => {
+            const { id, title } = tab;
+
+            return (
+              <li
+                key={id}
+                className={classNames({
+                  'is-active': selectedTabId === id || (!currentTab && !index),
```

Comment:
you repeated the check if current tab is selected 2 times
will be better to move this check to some variable before return statement in the map method, and use this variable where you need it